src/serac/physics/solid_mechanics.hpp

```diff
@@ -349,6 +349,16 @@ class SolidMechanics<order, dim, Parameters<parameter_space...>, std::integer_se
    * @tparam MaterialType The solid material type
    * @tparam StateType the type that contains the internal variables for MaterialType
    * @param material A material that provides a function to evaluate stress
+   * @pre body_force must be a object that can be called with the following arguments:
+   *    1. `tensor<T,dim> x` the spatial coordinates for the quadrature point
+   *    2. `double t` the time (note: time will be handled differently in the future)
+   *    3+. `tuple{value, derivative}`, a tuple of values and derivatives for each of the trial spaces
+   *            specified in the `DependsOn<...>` argument. 
+   *
+   * @note The actual types of these arguments passed will be `double`, `tensor<double, ... >` or tuples thereof
```

Comment:
This note is helpful. However, the arrow between the template classes is a little confusing, because it looks like broken C++. maybe replace it with "to" if we can't get an actual arrow glyph?

Reply:
This is font-dependent, a typeface with ligatures (e.g. Fira Code) does render an arrow glyph when the characters `->` appear adjacent.

Reply:
But I can still change it to something else if you prefer